commit f2dd96e607c6ecbeaf8a3d85e264a16a2c21a7e0
Author: Brian Norris <briannorris@chromium.org>
Date:   Fri Nov 2 10:17:47 2018 -0700

    ath10k: assign 'n_cipher_suites' for WCN3990
    
    [ Upstream commit 2bd345cd2bfc0bd44528896313c0b45f087bdf67 ]
    
    Commit 2ea9f12cefe4 ("ath10k: add new cipher suite support") added a new
    n_cipher_suites HW param with a fallback value and a warning log. Commit
    03a72288c546 ("ath10k: wmi: add hw params entry for wcn3990") later
    added WCN3990 HW entries, but it missed the n_cipher_suites.
    
    Rather than seeing this warning every boot
    
      ath10k_snoc 18800000.wifi: invalid hw_params.n_cipher_suites 0
    
    let's provide the appropriate value.
    
    Cc: Rakesh Pillai <pillair@qti.qualcomm.com>
    Cc: Govind Singh <govinds@qti.qualcomm.com>
    Signed-off-by: Brian Norris <briannorris@chromium.org>
    Signed-off-by: Kalle Valo <kvalo@codeaurora.org>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/drivers/net/wireless/ath/ath10k/core.c b/drivers/net/wireless/ath/ath10k/core.c
index c40cd129afe7..5210cffb5344 100644
--- a/drivers/net/wireless/ath/ath10k/core.c
+++ b/drivers/net/wireless/ath/ath10k/core.c
@@ -532,6 +532,7 @@ static const struct ath10k_hw_params ath10k_hw_params_list[] = {
 		.hw_ops = &wcn3990_ops,
 		.decap_align_bytes = 1,
 		.num_peers = TARGET_HL_10_TLV_NUM_PEERS,
+		.n_cipher_suites = 8,
 		.ast_skid_limit = TARGET_HL_10_TLV_AST_SKID_LIMIT,
 		.num_wds_entries = TARGET_HL_10_TLV_NUM_WDS_ENTRIES,
 		.target_64bit = true,
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1013_f2dd96e607c6ecbeaf8a3d85e264a16a2c21a7e0.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1013_f2dd96e607c6ecbeaf8a3d85e264a16a2c21a7e0.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
